lib/src/trim_editor.dart

```diff
@@ -516,4 +520,13 @@ class _TrimEditorState extends State<TrimEditor> with TickerProviderStateMixin {
       ),
     );
   }
+
+  _refreshVideoIfNeeded() async {
+    if (Platform.isIOS) {
+      await videoPlayerController.play();
+      Timer(_refreshVideoDuration, () {
```

Comment:
How did we come up with this magic 50ms number? What if on some devices it takes 51 or 52ms and this call fails?

Reply:
It's empiric. The ideal would-be if `seekTo` in `video_player` is consistent on `iOS` and `Android` and properly moves to a certain frame, but it doesn't, so we need to apply this hack. If you have a better solution in mind, go for it!
If for some reason this call fails, the behavior will be the same as in the old version (frame won't be refreshed).